validator/block_validator.go

```diff
@@ -409,7 +409,7 @@ func (v *BlockValidator) validate(ctx context.Context, validationStatus *validat
 		duration := time.Since(before)
 		if err != nil {
 			if !errors.Is(err, context.Canceled) && !errors.Is(err, context.DeadlineExceeded) {
```

Comment:
blockhash might be useful in case context.Canceled or DeadlineExceede.. maybe in these cases log Info or Warning instead of Error?